README.md

```diff
@@ -97,11 +97,11 @@ Examples:
 
 Adds a contact to the contact list.
 
-Format: `addc {name} e/ {email} p/ {phone number} [m/ {module1} {module2}] [t/ {tag1}]`
+Format: `addc n/ {name} p/ {phone_number} e/ {email} a/ {address} [t/ {tag}] [m/ {module1} [{module2} ]...] gh/ {github} tele/ {telegram}`
```

Comment:
should this be [m/ {module1} {module2} ...] ?

Reply:
Ey, that's a good one! The intention with this format was to specify that the space should be there, but I think the examples will make it clear enough.